payroll-processor-client/src/app/employee/employee-list/employee-list.component.ts

```diff
@@ -11,20 +15,23 @@ import { faSmileBeam, faSkull } from '@fortawesome/free-solid-svg-icons';
 export class EmployeeListComponent implements OnInit, OnDestroy {
   faSkull = faSkull;
   faSmileBeam = faSmileBeam;
+  employees = of<Employee[]>();
```

Comment:
If the query is assigned to the class field in the constructor (which I recommend) instead of `ngOnInit()`, then we don't need to pre-assign a value to the `employees` field, and we can mark it as `readonly`.

Reply:
fixed!